File: 240206/Next Level/next-level.py
class Introduce:
    def __init__(self, user_id = 'codetree', level = 10):
        self.user_id = user_id
        self.level = level


first = Introduce()
print(f"user {first.user_id} lv {first.level}")

i, l = input().split()
second = Introduce(i, l)
print(f"user {second.user_id} lv {second.level}")

File: 240812/Next Level/next-level.js
const fs = require("fs");
let [id, level] = fs.readFileSync(0).toString().trim().split(" ");

class UserInfo {
    constructor(id, level) {
        this.id = id;
        this.level = level;
    }
}

const user1 = new UserInfo("codetree", "10");
const user2 = new UserInfo();

user2.id = id;
user2.level = level;

console.log(`user ${user1.id} lv ${user1.level}`);
console.log(`user ${user2.id} lv ${user2.level}`);